projects/Draupnir/default.nix

```diff
@@ -32,5 +32,10 @@
         tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/matrix/draupnir.nix";
       };
     };
+    demo.vm = {
+      module = ./example.nix;
```

Comment:
There is no option to open the ports in the firewall, so we need to either create and enable that or add:

```nix
    # open service ports in the firewall
    networking.firewall.allowedTCPPorts = [ 8008 ];
```

In order for the synapse instance to be [forwarded to the host](https://github.com/ngi-nix/ngipkgs/blob/96154f436225554038061dd26d007e13a8b3d29a/overview/demo/vm/virtualisation.nix#L13-L19).